### DISCUSSION TITLE
Is it possible to make a modern beautiful background using axaml code?

### DISCUSSION THREAD

**Discussion by al-zakaria** [AvaloniaUI/Avalonia#17325] [Q&A]
for example, make a background consisting of two colors and there will be, for example, a triangle going from the middle of the left wall to the upper right corner, which will be another black and the rest of the background will be white

I don't know how to describe it, but I think everyone has seen this design

**Comment by StefanKoell** [comment]
You can do a lot in XAML or using custom rendering. There are also plenty of controls which allows you to do cool things, like this:
https://www.actiprosoftware.com/docs/controls/avalonia/shared/controls/mesh-gradient-presenter

**Reply by StefanKoell** [reply]
I'm not sure I completely understand. Do you mean how to use the control I mentioned? If so, check out:
https://www.actiprosoftware.com/docs/controls/avalonia/nuget

**Reply by al-zakaria** [reply]
Can I ask? Where do you find these libraries?

**Reply by al-zakaria** [reply]
and as I understand it, this is a paid product. is there a free alternative?

**Reply by StefanKoell** [reply]
It's both. Theme, Core and Shared is free (including what I linked). Fundamental and future products are paid.

Also check out this:
https://github.com/AvaloniaCommunity/awesome-avalonia
